cicd/2_prepare_rs_db.py

```diff
@@ -323,27 +299,24 @@ def process_database(
 
 
 if __name__ == "__main__":
-    # Update URL and connection method for your system
-    URL = "http://localhost/mi_servicelayer"
-    api_client = Connection(api_url=URL).with_autologon().connect()
-    gdl_session = gdl.GRANTA_MISession(url=URL, autoLogon=True)
+    api_client = Connection(api_url=MI_URL).with_autologon().connect()
+    gdl_session = gdl.GRANTA_MISession(url=MI_URL, autoLogon=True)
 
     logger.info("Loading saved RS information...")
+    INPUT_FILE_NAME = Path(__file__).parent / DATA_FILENAME
     if not INPUT_FILE_NAME.exists():
         logger.error("No saved data available. Run 'get_cleaned_db_entries.py' against an existing test DB first")
         exit(1)
     with open(INPUT_FILE_NAME, "r", encoding="utf8") as fp:
         input_data = json.load(fp)
 
-    table_browser = TableBrowser(api_client, logger)
+    table_browser = DatabaseBrowser(api_client, logger)
```

Comment:
Minor thing but the naming here seems inconsistent now, might be worth renaming the object too